samples/matter/light_switch/README.rst

```diff
@@ -43,8 +43,8 @@ IPv6 network support
 
 The development kits for this sample offer the following IPv6 network support for Matter:
 
-* Matter over Thread is supported for ``nrf52840dk_nrf52840``, ``nrf5340dk_nrf5340_cpuapp``, ``nrf21540dk_nrf52840``, and ``nrf54l15pdk_nrf54l15``.
```

Comment:
I noticed  there should be `nrf54l15pdk_nrf54l15_cpuapp` (it was wrong before your change). Could you fix this btw of making some next push? The same for the other Matter samples.